File: renewalMockKata/csharp/RenewalNotificationServiceTest/RenewalNotificationServiceTest.cs
using System;
using Xunit;

namespace Katas
{
    public class RenewalNotificationServiceTest
    {
        [Fact]
        public void notifyingAtRiskSubscribersShouldSendEmails()
        {
			RenewalNotificationService renewalNotificationService = new RenewalNotificationService(new RealSubscriberService(), null);
			renewalNotificationService.notifyAtRiskSubscribers();
			// Uh oh... how will we know what happened if the method doesn't return any information?
		}
    }
}

File: 06-RenewalMockKata/java/src/test/java/RenewalNotificationServiceTest.java
import org.junit.Test;

public class RenewalNotificationServiceTest {

    @Test
    public void notifyingAtRiskSubscribersShouldSendEmails() {
        RenewalNotificationService renewalNotificationService = new RenewalNotificationService(new RealSubscriberService(), null);
        renewalNotificationService.notifyAtRiskSubscribers();
//		Uh oh... how will we know what happened if the method doesn't return any information?
    }
}
